package.json

```diff
@@ -70,6 +70,7 @@
     "dompurify": "^2.2.9",
     "file-saver": "^2.0.5",
     "jquery": "^3.6.0",
+    "karma": "^6.3.11",
```

Comment:
why is this changing from a `devDependencies` to `dependencies`

Reply:
was trying a bunch of things to get follow-redirects to auto-update, fixed now